sbol_utilities/sbol3_genbank_conversion.py

```diff
@@ -339,83 +334,9 @@ def convert_sbol3_to_genbank(self, sbol3_file: str, doc: sbol3.Document = None,
                 self._reset_extra_properties_in_genbank(obj, seq_rec, references)
 
                 seq_rec_features = []
-                if obj.features:
```

Comment:
The extraction of this code block to a separate method is really great, and a nice step towards #168